dev/mocks.js

```diff
@@ -68,13 +68,12 @@ globalThis.Response = class Response {
   }
 }
 
-const chunk = (_, i, h) => (i % 2 ? [] : [[h[i], h[i + 1]]])
 globalThis.Request = class Request {
   constructor(req) {
-    this.headers = new Map(req.rawHeaders.flatMap(chunk))
+    this.headers = new Map(Object.entries(req.headers))
```

Comment:
we do not need to use rawHeaders anymore since it was only for supporting upper case headers